config/environments/development.rb

```diff
@@ -37,6 +37,9 @@
   config.action_mailer.raise_delivery_errors = false
 
   config.action_mailer.perform_caching = false
+  
+  host = 'localhost:3000'
+  config.action_mailer.default_url_options = { host: host, protocol: 'http' }
```

Comment:
```suggestion
  config.action_mailer.default_url_options = { host: 'localhost:3000', protocol: 'http' }
```
ri luôn cho gọn e, trong file config hạn chế đặt biến, vì config sẽ được load lên cùng với app

Reply:
dạ a